Project_Mars/Project_Mars/Pages/HomePage.cs

```diff
@@ -0,0 +1,30 @@
+﻿using OpenQA.Selenium;
+//using static Project_Mars.Utilities.CommonDriver;
+
+
+namespace Project_Mars.Pages
+{
+    public class HomePage
+    {
+        
+        public void SelectLanguageTab(IWebDriver driver)
```

Comment:
you have not implemented the POM pattern please implement this, see the link below for reference https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html

Reply:
please make use of the POM pattern in all pages

Reply:
![Homepage](https://github.com/Shilpii01/Project_Mars/assets/145233624/2e76eaf4-9dde-4f3d-9a2c-e36b74030c60)
Implemented the POM pattern